inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/stream/TemplateServiceImpl.java

```diff
@@ -106,6 +120,12 @@ public TemplateInfo get(String templateName, String operator) {
         TemplateInfo templateInfo = CommonBeanUtils.copyProperties(templateEntity, TemplateInfo::new);
         List<TemplateField> templateFields = getTemplateFields(templateEntity.getId());
         templateInfo.setFieldList(templateFields);
+        List<String> tenantList = tenantTemplateEntityMapper
```

Comment:
is it possible for `tenantTemplateEntityMapper.selectByTemplateName(templateName)` return null?

Reply:
Fixed.